iele.md

```diff
@@ -406,6 +406,129 @@ Some instructions require an argument to be interpreted as an address (modulo 16
     rule #addr?(REGS1 = call LABEL at W0 (REGS2) send W1 , gaslimit W2)     => REGS1 = call LABEL at #addr(W0) (REGS2) send W1 , gaslimit W2
     rule #addr?(REGS1 = staticcall LABEL at W0 (REGS2) gaslimit W1)         => REGS1 = staticcall LABEL at #addr(W0) (REGS2) gaslimit W1
     rule #addr?(OP)                                                         => OP [owise]
+```
+
+### Internal Operations
+
+-   `#loadAccount_` allows declaring a new empty account with the given address (and assumes the rounding to 160 bits has already occured).
+    If the account already exists with non-zero nonce or non-empty code, an exception is thrown.
+    Otherwise, if the account already exists, the storage is cleared.
+
+```{.k .uiuck .rvk .standalone .node}
+    syntax InternalOp ::= "#loadAccount" Int
+                        | "#initAccount" Int
+ // ----------------------------------------
+    rule <k> #initAccount ACCT => #exception ACCT_COLLISION ... </k>
+         <account>
+           <acctID> ACCT  </acctID>
+           <code>   CODE  </code>
+           <nonce>  NONCE </nonce>
+           ...
+         </account>
+      requires CODE =/=K #emptyCode orBool NONCE =/=K 0
+
+    rule <k> #initAccount ACCT => . ... </k>
+         <account>
+           <acctID>  ACCT       </acctID>
+           <code>    #emptyCode </code>
+           <nonce>   0          </nonce>
+           <storage> _ => .Map  </storage>
+           ...
+         </account>
+
+    rule <k> (.K => #loadAccount ACCT) ~> #initAccount ACCT ... </k>
+         <activeAccounts> ACCTS </activeAccounts>
+      requires notBool ACCT in ACCTS
+```
+
+```{.k .uiuck .rvk .standalone}
+    rule <k> #loadAccount ACCT => . ... </k>
+         <activeAccounts> ACCTS (.Set => SetItem(ACCT)) </activeAccounts>
+         <accounts>
+           ( .Bag
+          => <account>
```

Comment:
If the ACCT need to be created, should the balance, code, nonce cell be initialized?